ts/kpt-functions/src/types.ts

```diff
@@ -166,6 +166,29 @@ export class Configs {
     return this.functionConfig;
   }
 
+  /**
+   * Returns the functionConfig data field if defined.
+   */
+  getFunctionConfigDataOrThrow() {
```

Comment:
what's the return value? I think you want to convert to Map type.

Reply:
Done